doc/explanation/lxc_show_info.md

```diff
@@ -0,0 +1,49 @@
+(lxc-show-info)=
+# About `lxc` 'show' and 'info'
```

Comment:
```suggestion
# About `lxc` `show` and `info`

```

We display the sub-commands in the same way as the main command, thus in code style. That's a global change for this page.

Also, an empty line after the heading, just for consistency. :)